src/main/wallet/WalletScreen.tsx

```diff
@@ -1,50 +1,204 @@
 import React from 'react';
-import {
-  SafeAreaView,
-  StatusBar,
-  StyleSheet,
-  Text,
-  View,
-  useColorScheme,
-} from 'react-native';
-
-import { styledColors } from 'src/shared/styles';
 
 import { MainTabStackScreenProps } from '../MainStack';
+import {
+  CardSize,
+  QuaiPayCard,
+  QuaiPayContent,
+  QuaiPayListItem,
+  QuaiPaySearchbar,
+  QuaiPayText,
+} from 'src/shared/components';
+import { Pressable, ScrollView, StyleSheet, View } from 'react-native';
+import { Theme } from 'src/shared/types';
+import { useThemedStyle } from 'src/shared/hooks/useThemedStyle';
+import FilterIcon from 'src/shared/assets/filter.svg';
+import { useTranslation } from 'react-i18next';
+import { styledColors, typography } from 'src/shared/styles';
 
 const WalletScreen: React.FC<MainTabStackScreenProps<'Wallet'>> = ({}) => {
-  const isDarkMode = useColorScheme() === 'dark';
-
-  const backgroundStyle = {
-    backgroundColor: isDarkMode ? styledColors.black : styledColors.white,
-    width: '100%',
-    height: '100%',
-  };
+  const { t } = useTranslation();
+  const styles = useThemedStyle(themedStyle);
 
-  const topViewStyle = {
-    backgroundColor: isDarkMode ? styledColors.black : styledColors.white,
-    marginLeft: 10,
-    marginRight: 10,
-    marginTop: 'auto',
-    marginBottom: 'auto',
-  };
   return (
-    <SafeAreaView style={backgroundStyle}>
-      <StatusBar
-        barStyle={isDarkMode ? 'light-content' : 'dark-content'}
-        backgroundColor={backgroundStyle.backgroundColor}
-      />
-      <View style={topViewStyle}>
-        <View style={styles.switchStyle}>
-          <Text>Wallet Screen</Text>
-        </View>
+    <QuaiPayContent navButton={false}>
+      <View style={styles.cardWrapper}>
+        <QuaiPayCard
+          size={CardSize.Small}
+          quaiAmount="142.123"
+          address="0x123453.....0934823"
+          zone="Cyprus-02"
+          fiatAmount="1,000"
+          title={t('wallet.balance')}
+        />
+      </View>
+      <View style={styles.buttonWrapper}>
+        <Pressable
+          style={({ pressed }) => [
+            styles.button,
+            styles.addressButton,
+            pressed && { opacity: 0.5 },
+          ]}
+        >
+          <QuaiPayText style={[typography.H3, styles.colorOverwrite]}>
```

Comment:
you can pass the prop `type='H3'` rather than using the typography style

```suggestion
          <QuaiPayText type="H3" style={styles.colorOverwrite}>
```